tests/gtest/aviflosslesstest.cc

```diff
@@ -21,8 +21,12 @@ TEST(BasicTest, EncodeDecodeMatrixCoefficients) {
     const testutil::AvifImagePtr ground_truth_image =
         testutil::ReadImage(data_path, file_name);
 
-    for (auto matrix_coefficient :
-         {AVIF_MATRIX_COEFFICIENTS_IDENTITY, AVIF_MATRIX_COEFFICIENTS_YCGCO}) {
+    for (auto& matrix_coefficient : {
```

Comment:
Nit: It doesn't seem necessary to add `&`, because it is cheap to copy the `AVIF_MATRIX_COEFFICIENTS_xxx` values.

Reply:
Done